### ISSUE TITLE
Remote duck event bug (it is fired only once on a physical device)

### ISSUE THREAD

**Issue by matetami96** [doublesymmetry/react-native-track-player#1577]
**Describe the Bug**
When playing the contents of a .pls file which is used as a track, the remote-duck event is not fired correctly. Whenever something interrupts the player's music (incoming notification, a call etc.) the event is fired only once with the following values { permanent: true, paused: true }. 
I have an older version of the track player package (v1), the firing of the event works correctly on a virtual device but not on a physical device. Recently I have tried the new version of the track player (v2), here the event firing does not work correctly neither on a virtual device nor on a physical, I always get the following response { permanent: true, paused: true } and only once. Using version 1 of the track player with a virtual device I received two events with the following values: first { permanent: false, paused: true } then when the interruption ended { permanent: false, paused: false} and my code was able to react to these events without any problems.

**Steps To Reproduce**
To reproduce the bug create a React Native project that uses the react-native-track-player package, create a service.js as suggested by the documentation in the root folder, then follow the documentation for how to set up the track player for example in App.js. Then start the track player and try interrupting the continuous music, and check the logs. Use the following code that is provided below in service.js. Also use any kind of music file as a track I do not think that the .pls file's content (the URL) is causing the problem.

**Code To Reproduce**
```
import TrackPlayer from 'react-native-track-player';

module.exports = async function () {
  // monitor interrupt when other apps start playing music
  let wasPausedByDuck = false;

  TrackPlayer.addEventListener('remote-duck', async (e) => {
    console.log('The event: ', e);
    if (e.permanent === true) {
      TrackPlayer.stop();
    } else {
      if (e.paused === true) {
        const playerState = await TrackPlayer.getState();
        wasPausedByDuck = playerState !== TrackPlayer.STATE_PAUSED;
        TrackPlayer.pause();
      } else {
        if (wasPausedByDuck === true) {
          TrackPlayer.play();
          wasPausedByDuck = false;
        }
      }
    }
  });
}
```

**Environment Info:**
##### Paste the results of `npx react-native info`
```
System:
    OS: Windows 10 10.0.19044
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz       
    Memory: 12.03 GB / 23.92 GB
  Binaries:
    Node: 16.14.2 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.17.3 - C:\Program Files (x86)\Yarn\bin\yarn.CMD      
    npm: 8.5.0 - C:\Program Files\nodejs\npm.CMD
    Watchman: Not Found
  SDKs:
    Android SDK: Not Found
    Windows SDK:
      AllowAllTrustedApps: Disabled
      Versions: 10.0.17763.0
  IDEs:
    Android Studio: Version  4.1.0.0 AI-201.8743.12.41.7042882   
    Visual Studio: Not Found
  Languages:
    Java: 12.0.2 - C:\Program Files\Java\jdk-12.0.2\bin\javac.EXE
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.64.2 => 0.64.2 
    react-native-windows: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```
##### Paste the exact `react-native-track-player` version you are using
"react-native-track-player": "^1.2.3" (as mentioned above the new version had the same issue for me)
##### Real device? Or simulator?
For version 1 it did not work with a real device but it worked with a simulator.  
For version 2 it did not work with neither.
##### What OS are you running?
I am using Windows 10 Home version 21H2
**How I can Help**
##### What can you do to help resolve this?
I am sure most people have more experience with this package than I do, so please try reproducing the bug and if possible provide ways or tips on how to solve the issue. Also, let me know if the provided information is not enough or is confusing so I can edit it. 
##### Have you investigated the underlying JS or Swift/Android code causing this bug?
I have found the MusicManager.java file in the node modules that is supposed to fire the event but I do not have any Java knowledge so I am limited in my investigation of this bug.
##### Can you create a Pull Request with a fix?
No, I have not come up with any solutions to this problem yet.

**Comment by jspizziri** [comment]
@matetami96 `v1` is no longer supported. If you have an issue with version 2 please create a new issue including reproduction steps (ideally ones based on the `example` project).

Thanks!
